packages/create-webstone-app/src/tasks/3-configure-app/plugin.ts

```diff
@@ -0,0 +1,73 @@
+import { execa } from "execa";
+import fs from "fs-extra";
+import { ListrTask } from "listr2";
+import path, { dirname } from "path";
+import { fileURLToPath } from "url";
+import { Ctx, determinePackageManager } from "../../helpers";
+import json5 from "json5";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export const installDeps = async (ctx: Ctx) => {
+  const installProcess = execa(
+    determinePackageManager(),
+    ["add", "-D", "@webstone/cli", "@webstone/gluegun"],
+    {
+      shell: true,
+      cwd: ctx.appDir,
+    }
+  );
+
+  return installProcess.stdout;
+};
+
+export const copyFiles = async (ctx: Ctx) => {
+  const tsconfigPath = path.join(
+    __dirname,
+    "..",
+    "template",
+    "plugin",
+    "tsconfig.json"
+  );
+  const cliPath = path.join(__dirname, "..", "template", "plugin", "cli");
+  fs.copySync(tsconfigPath, `${ctx.appDir}/tsconfig.cli.json`);
+  fs.copySync(cliPath, `${ctx.appDir}/src/lib/cli`);
+};
+
+export const adjustConfigFiles = async (ctx: Ctx) => {
+  //package.json
+  const packageJson = await fs.readJSON(`${ctx.appDir}/package.json`);
+  packageJson.scripts.build = `${packageJson.scripts.build} && tsc -p tsconfig.cli.json`;
+  await fs.writeJSON(`${ctx.appDir}/package.json`, packageJson, { spaces: 2 });
+
+  //tsconfig
+  const tsconfig = await fs.readFile(`${ctx.appDir}/tsconfig.json`, "utf-8");
+
+  const tsconfigJson = json5.parse(tsconfig);
```

Comment:
We may want to look if its possible to keep comments after parsing json5, but I think for this PR it's not a blocker 🤔

Reply:
I agree, not a blocker for now.